modules/web-console-cso.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * web_console/scan-from-web-console.adoc
+
+[id="web-console-cso_{context}"]
+= Run Container Security Operator in {product-title}
+
+To start the Container Security Operator, go to the {product-title}
+web console, then select and install that Operator from the Operator Hub,
+as described here.
+
+.Prerequisites
+
+* Be running {product-title} 4.2 or later
```

Comment:
I don't think this is necessary. This section should only appear in the docs for the versions that it's appropriate for. Plus 4.1 isn't even supported anymore, so that shouldn't be an issue.

Reply:
Done